src/test/java/core/basesyntax/service/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.FileReaderService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderServiceImplTest {
+    private static final String PATH_DATA_FILE = "src/test/java/resources/dataTest.csv";
+    private static final String PATH_EMPTY_FILE = "src/test/java/resources/emptyTest.csv";
+    private static FileReaderService fileReaderService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fileReaderService = new FileReaderServiceImpl();
+    }
+
+    @Test
+    public void readFile_validData_ok() {
+        final List<String> actual = fileReaderService.readFile(PATH_DATA_FILE);
+        List<String> expected = new ArrayList<>();
+        expected.add("testing");
+        expected.add("readFile_Ok method,");
+        expected.add("last line.");
+        assertEquals(expected, actual);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void readFile_emptyData_notOk() {
+        List<String> actual = fileReaderService.readFile(PATH_EMPTY_FILE);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void readFile_pathIsNull_notOk() {
+        List<String> actual = fileReaderService.readFile(null);
```

Comment:
What exception will occur if you pass a null value to the method ```readFile```? You don’t need test system exceptions, so add checking to your solution or remove this test case